client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import methodShape from '../../../types/registeredMethod';
+import ReadOnlyView from './ReadOnlyView';
+
+class RegisteredMFAMethodListField extends Component {
+  /**
+   * The backup and default methods are rendered separately
+   * @returns {*}
+   */
+  baseMethods() {
+    const { backupMethod, defaultMethod } = this.props;
+    let methods = this.props.registeredMethods;
+
+    if (backupMethod) {
+      methods = methods.filter(rM => rM.urlSegment !== backupMethod.urlSegment);
+    }
+
+    if (defaultMethod) {
+      methods = methods.filter(rM => rM.urlSegment !== defaultMethod.urlSegment);
+    }
+
+    return methods;
+  }
+
+  render() {
+    const { defaultMethod, readOnly } = this.props;
+
+    if (readOnly) {
+      return (
+        <ReadOnlyView
+          defaultMethod={defaultMethod}
+          methods={this.baseMethods()}
+        />
+      );
+    }
+
+    return (<h1>(EditableView)</h1>);
```

Comment:
What's this?

Reply:
It was a placeholder for the read/write UI which will come late. I've refactored this to just render the read-only state in all cases for now.